src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -134,10 +150,10 @@ export async function stop(appName: string): Promise<void> {
 
 export async function pause(appName: string): Promise<void> {
   const dl = inProgressDownloadsMap.get(appName)
-  if (dl === undefined) {
+  if (dl === undefined || dl.DownloadItem === undefined) {
```

Comment:
```suggestion
  if (!dl?.DownloadItem) {
```

Perhaps this is simpler?